src/Misc/layoutbin/powershell/Add-JavaCapabilities.ps1

```diff
@@ -17,6 +24,19 @@ $null = Add-CapabilityFromRegistry -Name 'java_8' -Hive 'LocalMachine' -View 'Re
 $null = Add-CapabilityFromRegistry -Name 'java_6_x64' -Hive 'LocalMachine' -View 'Registry64' -KeyName $jre6KeyName -ValueName 'JavaHome' -Value ([ref]$latestJre)
 $null = Add-CapabilityFromRegistry -Name 'java_7_x64' -Hive 'LocalMachine' -View 'Registry64' -KeyName $jre7KeyName -ValueName 'JavaHome' -Value ([ref]$latestJre)
 $null = Add-CapabilityFromRegistry -Name 'java_8_x64' -Hive 'LocalMachine' -View 'Registry64' -KeyName $jre8KeyName -ValueName 'JavaHome' -Value ([ref]$latestJre)
+
+if  (-not (Test-Path env:DISABLE_JAVA_CAPABILITY_HIGHER_THAN_9)) {
```

Comment:
Test-Path with empty string as argument throws exception

Reply:
I assume you mean this case from PS documentation. If so it is about the null value that is passed as Path argument, in our case the value of the argument is always corresponding to string env:DISABLE_JAVA_CAPABILITY_HIGHER_THAN_9 .
You can try to execute Test-Path command with the not existing env variable and you will see that in this case, it will return false, not throw an exception.